docs/product-installation-guide/src/main/asciidoc/clustering-elasticsearch-proc.adoc

```diff
@@ -1,34 +1,27 @@
 [id='clustering-elasticsearch-proc_{context}']
 = Setting up Elasticsearch
-//COMMENT: Will elasticseach be productized in time for PAM 7.1?
-Elasticsearch is a highly scalable open source full-text search and analytics engine. It enables you to store, search, and analyze high volumes of data quickly and in near real time.  In a {PRODUCT} clustered environment, it enables you to perform complex and efficient searches across cluster nodes. Set up Elasticsearch on one node of the cluster.
+Elasticsearch is a highly scalable open source full-text search and analytics engine. It enables you to store, search, and analyze high volumes of data quickly and in near real time.  In a {PRODUCT} clustered environment, it enables you to perform complex and efficient searches across cluster nodes. 
 
-.Prerequisite
-A {EAP} 7.1 cluster is configured as described in  https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/configuring_a_red_hat_jboss_eap_7_cluster/[_Configuring a {EAP} 7 Cluster_].
+Set up Elasticsearch on one node of the cluster.
+
+[NOTE]
+====
+This procedure describes how to set up Elasticseach for a simplified, non-high availability environement. For high availability production environments, see the https://www.elastic.co/guide/en/elasticsearch/guide/current/deploy.html[Production Deployment] section of the Elasticsearch documentation.
```

Comment:
Super trivial (comes up a couple of times): "non-high availability" should actually use an en dash instead of a high (hyphens with compound words in Chicago), but I haven't seen how to easily do it yet in Atom (doesn't come up enough). Em dashes are fine in Atom, but no en dashes that I see yet.